src/contracts/entrypoints.rs

```diff
@@ -12,22 +12,63 @@ use ink::primitives::AccountId;
 
 #[ink::trait_definition]
 pub trait Invariant {
+    /// Retrieves the protocol fee represented as a percentage.
     #[ink(message)]
     fn get_protocol_fee(&self) -> Percentage;
 
+    /// Allows an authorized user to withdraw collected fees.
+    ///
+    /// # Parameters
+    /// - `pool_key`: A unique key that identifies the specified pool.
+    ///
+    /// # Errors
+    /// - Reverts the call when the caller is an unauthorized receiver.
     #[ink(message)]
     fn withdraw_protocol_fee(&mut self, pool_key: PoolKey) -> Result<(), InvariantError>;
 
+    /// Allows an authorized user to adjust the protocol fee.
+    ///
+    /// # Parameters
+    /// - `protocol_fee`: The expected fee represented as a percentage.
+    ///
+    /// # Errors
+    /// - Reverts the call when the caller is an unauthorized user.
     #[ink(message)]
     fn change_protocol_fee(&mut self, protocol_fee: Percentage) -> Result<(), InvariantError>;
 
+    /// Transfers fee receiver authorization to another user.
```

Comment:
people reading that might think that fee receiver can also transfer their role to another user, you should be more specific